packages/cli/src/commands/new/new.ts

```diff
@@ -70,25 +66,72 @@ export default async (opts: OptionValues) => {
     return dir;
   }
 
-  const license = opts.license ?? 'Apache-2.0';
+  const dirName = createDirName(template, options);
+  const targetDir = paths.resolveTargetRoot(options.targetPath, dirName);
+
+  const packageName = resolvePackageName({
+    baseName: dirName,
+    scope: options.scope,
+    plugin: template.plugin ?? true,
+  });
+
+  const moduleVar =
+    options.moduleId ??
+    `${camelCase(options.id)}Module${camelCase(
+      options.moduleId,
+    )[0].toUpperCase()}${camelCase(options.moduleId).slice(1)}`; // used in default-backend-module template
+  const extensionName = `${upperFirst(camelCase(options.id))}Page`; // used in default-plugin template
+  const pluginVar = `${camelCase(options.id)}Plugin`; // used in default-backend-plugin and default-plugin template
```

Comment:
What do you think of having some form of helper to help do this in the template instead? Something like `{{ prompts.pluginId + '-plugin' | camelCase }}`.

We have been talking about moving the scaffolder to https://liquidjs.com/, and could potentially start with these templates. Happy to leave that as a follow-up though